Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM plus3it/tardigrade-ci:0.0.4
+FROM plus3it/tardigrade-ci:0.0.6
```

Comment:
doublecheck that you are based on upstream/master... this should really be handled by dependabot...